client/me/help/help-contact/index.jsx

```diff
@@ -294,7 +294,7 @@ class HelpContact extends React.Component {
 		if ( site || userDeclaredUrl ) {
 			const siteUrl = userDeclaredUrl ? userDeclaredUrl.trim() : site.URL;
 
-			blogHelpMessage = this.translateForForums( 'Site: %s.', {
+			blogHelpMessage = this.translateForForums( 'Site: %s', {
 				args: [ userRequestsHidingUrl ? 'help@' + withoutHttp( siteUrl ) : siteUrl ],
```

Comment:
I know this line is not this PR related but from the history @Aurorum it seems you added it, right now it seems to be a globally declared function because I can't find the import, is this intended?

I found an import in `client/components/clipboard-button-input/index.jsx` maybe it could be helpful